.circleci/config.yml

```diff
@@ -22,6 +22,11 @@ commands:
                     pattern: ".*rc[0-9]+$"
                     value: ${CIRCLE_TAG}
           steps:
+            - run:
+                name: Import GPG key
+                command: |
+                  echo -e "$GPG_1X_PRIVATE_KEY" > private.key
+                  gpg --batch --import private.key
```

Comment:
this looks good, but it looks like it is being imported as a file into the build environment at the same level as kappa's folder, are we sure it won't leak into packages/containers that are released

Reply:
Could we add a step to `rm` the file, or pipe the key directly to `gpg`?